suites/quincy/nvmeof/tier-1_nvmeof_functional_Regression.yaml

```diff
@@ -109,23 +109,23 @@ tests:
         rep_pool_config:
           pool: rbd
         install: false                           # Run SPDK with all pre-requisites
-        subsystems: # Configure subsystems with all sub-entities
+        subsystems:                             # Configure subsystems with all sub-entities
           - nqn: nqn.2016-06.io.spdk:cnode2
             serial: 2
             bdevs:
               count: 1
               size: 10G
             listener_port: 5002
             allow_host: "*"
-        initiator: # Configure Initiators with all pre-req
-            subnqn: nqn.2016-06.io.spdk:cnode2
+        initiators:                             # Configure Initiators with all pre-req
+          - subnqn: nqn.2016-06.io.spdk:cnode2
             listener_port: 5002
             node: node7
-      desc: Perform Data integrity test over NVMEoF targets
+      desc: Deploy NVMEoF Gateway collocated with OSD node
       destroy-cluster: false
-      module: test_ceph_nvmeof_data_integrity.py
-      name: Test to perform Data Integrity test over NVMe-OF targets
-      polarion-id: CEPH-83576094
+      module: test_ceph_nvmeof_gateway.py
+      name: Test to setup NVMEoF Gateway collocated with OSD node and run IOs
```

Comment:
@preethisn since we are not deploying GW newly(install is false here and GW is already deployed on node6 in previous test) with OSD node in this test case, can we name it something else with its purpose?